projects/extension/public/manifest.json

```diff
@@ -4,14 +4,13 @@
   "homepage_url": "https://github.com/paritytech/substrate-connect",
   "name": "Substrate Connect Extension",
   "short_name": "substrate-connect",
-  "version": "0.2.0",
-  "manifest_version": 3,
+  "version": "0.2.1",
+  "manifest_version": 2,
   "permissions": ["notifications", "storage", "tabs", "alarms"],
   "background": {
-    "service_worker": "background.js",
-    "type": "module"
+    "scripts": ["background.js"]
```

Comment:
As I've mentioned, you can leave the `service_worker` if it works

Reply:
It is not working with `service_worker` and manifest v2